src/printer.rs

```diff
@@ -89,6 +97,8 @@ impl<'a> Printer<'a> {
                 .join(" ")
         )?;
 
+        self.handle.flush()?;
```

Comment:
This does not really work yet. I'm afraid we have to drop the lock on stdout to flush(?).

Reply:
Oh, this appeared as if it didn't work because the output was captured by `less`. Everything works fine if I use `--paging=never`.

I will probably switch of paging if stdin is used and `isatty(stdin) == true`.